src/proto/factories/layer_graph_factory.cpp

```diff
@@ -188,10 +193,6 @@ construct_layer_graph(lbann_comm* comm,
     ps.depth_groups = proto_layer.parallel_strategy().depth_groups();
     ps.depth_splits = proto_layer.parallel_strategy().depth_splits();
 
-    // Check that layer has been constructed
-    if (l == nullptr)
-      LBANN_ERROR("Could not construct layer ", name);
-
```

Comment:
Might want to check that a second time here too, just in case.

Reply:
Why? The pointer can't change at this point.